app/components/global/AppBar/AppBar.js

```diff
@@ -71,4 +70,18 @@ const AppBar = ({ user, menuItems }) => (
   </AppBarContainer>
 )
 
+AppBar.propTypes = {
+  menuItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string.isRequired,
+      link: PropTypes.string.isRequired,
+    }),
+  ).isRequired,
+  user: PropTypes.shape({}),
```

Comment:
`PropTypes.object`?

Perhaps it would also be an idea to add PropTypes to the `ProfileMenu` component within this PR, for greater specificity on what shape this object needs to take?

Reply:
We're using the `react/forbid-prop-types` linting rule which prevents `any` `array` and `object` as propTypes. I suppose I should be looking at the user values being applied within the `ProfileMenu` component and defining the minimum required attributes of the object if it exists here.